solr/core/src/test-files/solr/configsets/upload/with-lib-directive/solrconfig.xml

```diff
@@ -37,8 +37,6 @@
 
   <luceneMatchVersion>${tests.luceneMatchVersion:LATEST}</luceneMatchVersion>
 
-  <lib dir="${solr.install.dir:../../../..}/modules/ltr/lib/" regex=".*\.jar" />
```

Comment:
Maybe this file doesn't need to exist at all?

Reply:
Good catch - removed!